Added Feature Flags

JustMeDevin opened the pull request:
Added ability to request feature flags from device

JustMeDevin committed:
Added Feature Flags

Added ability to request feature flags from device

jmaha commented on Sources/CombustionBLE/BleData/FeatureFlags.swift:
```diff
@@ -0,0 +1,57 @@
+    
+    let wifi: Bool
+    
+    static let MASK: UInt8 = 0x0001
```
Since there will be a number of feature flags, it might be good to name this `WIFI_MASK`.

jmaha reviewed (approved):
Looks great! One small comment.

JustMeDevin committed:
Added version check

Added a check to not request feature flags if device running old firmware

JustMeDevin committed:
Fix for incorrect serial number against meatnode

Fixed setting incorrect serial number for meat net node, fixed feature flag response not being caught correctly

JustMeDevin committed:
Fix for wifi indexing

Fix for using incorrect index for wifi flag
